src/main/clojure/clara/rules.cljc

```diff
@@ -48,13 +48,16 @@
   "Runs the given query with the optional given parameters against the session.
    The optional parameters should be in map form. For example, a query call might be:
 
-   (query session get-by-last-name :last-name \"Jones\")
+   (query session get-by-last-name :?last-name \"Jones\")
```

Comment:
This was just incorrect docs correct? ie. not a change specific to this PR changes?

Reply:
yes, the docs were incorrect, so I fixed it.

Reply:
I also considered that with a few changes we could make it possible to pass parameters as :last-name and have the keyword be converted to `:?last-name`, but that would probably add additional complexity that's just not needed. If that's something useful we can add it in the future.